src/plugins/dump/dump.cpp

```diff
@@ -528,6 +587,7 @@ int elektraDumpGet (ckdb::Plugin * handle, ckdb::KeySet * returned, ckdb::Key *
 	keyDel (root);
 	int errnosave = errno;
 
+	// TODO: document
```

Comment:
TODO, is this needed for the new version of plugin process, too? (remove if not)

Reply:
I will remove this with the removal of the old `process` and renaming of `stdioproc` (after PR reviews are done).